src/specific/s_filesystem.c

```diff
@@ -35,3 +58,71 @@ void S_File_CreateDirectory(const char *path)
     mkdir(path, 0664);
 #endif
 }
+
+// r must have strlen(path) + 2 bytes
+bool S_File_CasePath(char const *path, char *r)
+{
+    size_t length = strlen(path);
+    char *p = Memory_Alloc(length + 1);
+    strcpy(p, path);
+    size_t rl = 0;
+
+    DIR *d;
+    if (p[0] == '/') {
+        d = opendir("/");
+        p = p + 1;
+    } else {
+        d = opendir(".");
+        r[0] = '.';
+        r[1] = 0;
+        rl = 1;
+    }
+    LOG_DEBUG("r: %s", r);
```

Comment:
Unnecessary debug calls, the one in `File_Open` can stay but these should go away in the final version

Reply:
My bad. Testing relics.